docs/sources/administration/team-management/configure-grafana-teams.md

```diff
@@ -0,0 +1,213 @@
+1. Click the arrow next to **Administration** in the left-side menu, click **Users and access**, and select **Teams**. 
+1. Click **New Team**.
+1. Fill in each field and click **Create**.
+1. Click **Save**. You can now add a Team member.
+
+## Add a Team member
+
+Add a member to a new Team or add a team member to an existing Team when you want to provide access to team dashboards and folders to another user. This task requires that you have `organization administrator` permissions.
+
+To add a team member, complete the following steps:
+
+1. Sign in to Grafana as an `org administrator` or `team administrator`.
+1. Click the arrow next to **Administration** in the left-side menu, click **Users and access**, and select **Teams**. 
+1. Click the name of the Team to which you want to add members, and click **+ Add member**.
+1. Search for and select a user.
+1. Choose whether to add the user as a Team **Member** or **Admin**.
+1. Click **Save**.
+
+## Grant or change Team member permissions
+
+Complete this task when you want to add or modify team member permissions.
+
+To grant team member permissions:
+
+1. Sign in to Grafana as an `org administrator` or `team administrator`.
+1. Click the arrow next to **Administration** in the left-side menu, click **Users and access**, and select **Teams**. 
+1. Click the pencil next to the name of the Team for which you want to add or modify team member permissions.
+1. In the team member list, locate the user that you want to change. You can use the search field to filter the list if necessary.
+1. Under the **Permission** column, select the new permission level. 
+
+## Add roles to a Grafana Team
+
+You can add or delete roles from a specified team.  
+
+To add a role, complete the following steps:
+
+1. Sign in to Grafana as an `org administrator` or `team administrator`.
+1. Click the arrow next to **Administration** in the left-side menu, click **Users and access**, and select **Teams**. 
+1. Select the Team and click under the **Role** column. Select from a list of current fixed or plugin roles or clear all roles and start over. As you hover over each role a list of permissions appears to the right. You can uncheck any permission for additional fine-grained control.
+1. Click **Update** to add the new role or roles. 
+
+To delete a role, remove the check next to the role name and click **Update**. 
+
+## Delete a team
+
+Delete a team when you no longer need it. This action permanently deletes the team and removes all team permissions from dashboards and folders. This task requires that you have `organization administrator` permissions.
+
+1. Sign in to Grafana as an `org administrator` or `team administrator`.
+1. Click the arrow next to **Administration** in the left-side menu, click **Users and access**, and select **Teams**. 
+1. Click the **red X** on the right side of the name of the team.
+1. Click **Delete**.
+
+## Create isolated or collaborative teams
```

Comment:
I would remove the Create isolated or collaborative teams section for now.
Going to work with @IevaVasiljeva to prepare an example in terraform that should be easier to follow than this code snipet.

Reply:
@sarahleejane I can comment this out for now.

Reply:
done